src/services/user-service.js

```diff
@@ -8,7 +8,7 @@ import { SAVE_USER_RESULT } from '../domain/user-management.js';
 import PasswordResetRequestStore from '../stores/password-reset-request-store.js';
 
 const DEFAULT_ROLE_NAME = ROLE.user;
-const PROVIDER_NAME_ELMU = 'elmu';
+const PROVIDER_NAME_EDUCANDU = 'elmu';
```

Comment:
I think this should not change, or? It's getting confusing.

Reply:
We need to make these configurable / client. I left the value elmu because we don't have that yet, but the provider name will probably have to be configurable for each client.
My bigger problem with this one was if to rename it to this or EDUCANDU_PROVIDER_NAME

Reply:
Then just `PROVIDER_NAME`, since it will be configurable?

Reply:
In the end, I think this should be "educandu" as it is the built-in auth provider. Opposed to "shibboleth" etc.

Reply:
But we need a migration for that

Reply:
Renamed to provider name so far since I don't know what this entails.